drive/types.bal

```diff
@@ -404,6 +405,7 @@ public type FilesResponse record {
 public type GetFileOptional record {
```

Comment:
If we use this record internally only, shall we not make it public?

Reply:
Yeah. It's only used internally. We can change to `public`.
Changed it all places.
/Thanks